FirebasePerformance/Tests/Unit/FPRSessionDetailsTest.m

```diff
@@ -29,6 +29,17 @@ - (void)testInstanceCreation {
   XCTAssertNotNil(details);
 }
 
+/** Validates object copy contains same details as the source. */
+- (void)testInstanceCopy {
+  FPRSessionDetails *details = [[FPRSessionDetails alloc] initWithSessionId:@"random"
+                                                                    options:FPRSessionOptionsNone];
+  FPRSessionDetails *detailsCopy = [details copy];
```

Comment:
Just to make sure that calling `copy` is equivalent to call `copyWithZone`?

Reply:
Yes, they are. `Copy` internally calls copying with the zone and the platforms associates the copy with the default zone.